### ISSUE TITLE
[How-To] Quantize my own model in Tensorflow using this approach?

### ISSUE THREAD

**Issue by mahimairaja** [jundaf2/INT8-Flash-Attention-FMHA-Quantization#4]

**Comment by jundaf2** [comment]
This dynamic quantization simply leverage the fact that 
(1) `the global max of the matrix P is not necessarily the max value of each line of matrix P`
(2)`you know the max and min value in the softmax computation of a line on the fly -- it's the inherit property of softmax, i.e. the numerators in every line  is between [0,1]`
, so you can leverage this fact without passing addtional global quantization information of matrix $P$.

Thanks.
